conda-store-server/conda_store_server/_internal/worker/build.py

```diff
@@ -243,14 +245,12 @@ def build_conda_environment(db: Session, conda_store, build):
                 build.id,
                 build.conda_lock_key,
                 json.dumps(
-                    context.result, indent=4, cls=utils.CustomJSONEncoder
+                    conda_lock_spec, indent=4, cls=utils.CustomJSONEncoder
```

Comment:
I know you didn't add this, but why are we indenting stuff in the DB? Seems like a waste of space.